ui/src/saros/ui/browser/IBrowserWrapper.java

```diff
@@ -0,0 +1,78 @@
+package saros.ui.browser;
+
+public interface IBrowserWrapper {
```

Comment:
What is the problem with IBrowser ?

Reply:
You mean the name or the interface of the swt-browser? if you mean the name: sure we can call it IBrowser.

Reply:
The name of the interface. I cannot remember that I have seen an IXyzWrapper in any application yet.

Reply:
The postfix `Wrapper` is not very informative. I will remove it.